cmd/plugin/builder/command/publish/helper_test.go

```diff
@@ -66,6 +66,7 @@ func convertVersionToStringArray(arrOsArchInfo []osArch) []string {
 	return oa
 }
 
+//nolint:unparam
```

Comment:
I have disabled because of the error  `cmd/plugin/builder/command/publish/helper_test.go:69:48: `createDummyArtifactDir` - `pluginName` always receives `"fake-plugin-foo"` (unparam)
`